deep_agent/aegra/lifecycle.py

```
@@ -0,0 +1,680 @@
+
+    try:
+        pattern = f"{REDIS_INFLIGHT_PREFIX}*"
+        keys = list(client.scan_iter(match=pattern, count=100))
+        run_ids = []
+        for key in keys:
+            data = client.hgetall(key)
+            if data.get("pod_id") == POD_ID:
+                rid = data.get("run_id", "")
+                if rid:
+                    run_ids.append(rid)
+        return run_ids
```

Ruff (0.15.21)

[warning] Consider moving this statement to an `else` block

(TRY300)